lib/ansible/module_utils/urls.py

```diff
@@ -840,22 +848,28 @@ def fetch_url(module, url, data=None, headers=None, method=None,
         info.update(r.info())
         info['url'] = r.geturl()  # The URL goes in too, because of redirects.
         info.update(dict(msg="OK (%s bytes)" % r.headers.get('Content-Length', 'unknown'), status=200))
```

Comment:
Add one ziploader comment around here so that we go through all of these and convert them over when the time comes.